src/ostorlab/agent/mixins/agent_persist_mixin.py

```diff
@@ -225,3 +225,39 @@ def add_ip_network(
         else:
             member_value = ip_range.exploded
         return self.set_add(key, member_value)
+
+    def ip_network_exists(
```

Comment:
The prefix "is_" suggests that the function returns a boolean value, which can enhance readability and make the purpose of the function more explicit.
```suggestion
    def is_ip_network_exists(
```

Reply:
Convention is `is_blue`, `is_red` and `file_exists`. Not file `is_file_exists`.